lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -752,5 +928,37 @@ inline bool ProxyRenderDelegate::ProxyShapeData::IsExcludePrimsUpToDate() const
 inline void ProxyRenderDelegate::ProxyShapeData::ExcludePrimsUpdated() {
     _excludePrimsVersion = _proxyShape->getExcludePrimPathsVersion();
 }
+inline void ProxyRenderDelegate::ProxyShapeData::UpdatePurpose(
```

Comment:
Do we want this code to be inline?

Reply:
We only call it once so it's not bloating the code right now.